programmers2/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+    const arr = s.split("");
+    if (arr.length % 2 === 0) {
+        const startNum = (arr.length / 2);
+        return arr[startNum - 1] + arr[startNum];
+    }
```

Comment:
여러번 쓰이는 arr.length 는 변수 선언을 하는것이 좋아보임.
=== 0 은 생략가능 하다.
```suggestion
    if (arr.length % 2) {
        const startNum = (arr.length / 2);
        return arr[startNum - 1] + arr[startNum];
    }
```